actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -123,9 +131,16 @@ def remove_channel(channel)
             @queue.push([:unlisten, channel])
           end
 
-          def invoke_callback(*)
+          def invoke_callback(callback, message)
+            message = decoder.decode(message)
```

Comment:
We should move decoding into the `#broadcast` method to decompress once; the `#invoke_callback` is called per subscribed client.